Getting-Started-Developing-on-ev3dev.md

```diff
@@ -209,6 +209,60 @@ Since this is a huge topic in itself, this guide will not get into further detai
 
 # Downloading Programs to Target
 
+* Setting Up SSH Access
+* 
+*
+*
+
+## Setting up SSH Access
+
+Secure Shell (SSH) is the recommended way to connect to the Robot Controller from the Host. The most commonly used SSH package is OpenSSH, which is supported by all the major OSes. Alternatively, Putty is available on Windows as a SSH client.
+
+> The default Developer's Firmware from LEGO suggests using `telnet` for access to the EV3 Robot Controller. Resist all temptations to use `telnet`, as it is an obsolete remote access client which transmits *EVERYTHING* you type in cleartext over the network link. You should excise any usage of `telnet` from your repertoire of command line tools as it is the road to grief and security compromises.
```

Comment:
FWIW, `telnet` is the only way to connect to the EV3 when running the official LEGO firmware and it only works with the "D" version of the firmware (D is for developer). Before version 1.09 of the firmware, passwordless root access was available via telnet and there is at least one [documented (in German) case](https://translate.google.com/translate?sl=auto&tl=en&js=y&prev=_t&hl=en&ie=UTF-8&u=http%3A%2F%2Fwww.heise.de%2Fsecurity%2Fartikel%2FAnalysiert-Lego-Mindstorms-fuer-Cyber-Angriffe-missbraucht-3055305.html&edit-text=&act=url) of an EV3 being compromised.

Reply:
Also, FWIW, telent is not installed on ev3dev by default, so it is not even an option.

Reply:
I've reworded the paragraph slightly.
Thanks for the link. I've added it as an example of security compromise.